src/EnergyPlus/SolarShading.cc

```diff
@@ -1334,6 +1334,12 @@ namespace SolarShading {
 		++NumAnisoSky_Calls;
 #endif
 
+		Real64 curSunlitFrac = 0.;
+		Real64 CurDifShdRadioHoriz = 0.;
+		if ( TimeStep == 4 && HourOfDay == 9 && DayOfSim == 167 ){
+			int x = 1;
+		}
```

Comment:
Just a debugging aid. 

I put a breakpoint on the "int x=1;" line and it is an easy way to get the simulation to stop at a certain time. I have tried using the hit count breakpoints but because how EnergyPlus iterates, it doesn't always come out to be the same time for different (but similar) test files.